clorinde/src/cli.rs

```diff
@@ -9,77 +9,110 @@ use crate::{config::Config, conn, container, error::Error, gen_live, gen_managed
 struct Args {
     #[clap(subcommand)]
     action: Action,
-
-    /// Config file path
-    #[clap(short, long, default_value = "clorinde.toml")]
-    config: PathBuf,
-
-    /// Use `podman` instead of `docker`
-    #[clap(short, long)]
-    podman: bool,
-    /// Folder containing the queries
-    #[clap(short, long, default_value = "queries/")]
-    queries_path: PathBuf,
-    /// Destination folder for generated modules
-    #[clap(short, long, default_value = "clorinde")]
-    destination: PathBuf,
-    /// Generate synchronous rust code
-    #[clap(long)]
-    sync: bool,
-    /// Generate asynchronous rust code
-    #[clap(long)]
-    r#async: bool,
-    /// Derive serde's `Serialize` trait for generated types.
-    #[clap(long)]
-    serialize: bool,
 }
 
 #[derive(Debug, Subcommand)]
 enum Action {
     /// Generate your modules against your own db
     Live {
+        #[clap(env = "DATABASE_URL")]
```

Comment:
Nice, didn't know about using `env` with clap. This will be useful in the future.